apps/ledger-live-desktop/src/renderer/components/EntryButton/EntryButton.jsx

```diff
@@ -0,0 +1,147 @@
+// @flow
+
+import styled from "styled-components";
+import Text from "~/renderer/components/Text";
+import { focusedShadowStyle } from "~/renderer/components/Box/Tabbable";
+import Box from "~/renderer/components/Box";
+import React from "react";
+import ChevronRight from "~/renderer/icons/ChevronRight";
+
+const TitleText = styled(Text).attrs(() => ({
```

Comment:
Can we use existing "typography"?

Reply:
https://react-ui-storybook.vercel.app/?path=/story/asorted-typography-text--overview

Reply:
I have been using this component `apps/ledger-live-desktop/src/renderer/components/Text.js` (which is used in lots of places in LLD) - and it doesn't seem to take in a `variant` prop. I'll try to use the Text component from react-ui instead.

Reply:
@sarneijim changed - now using `Text` from react-ui